science/ecCodes/files/patch-no-jasper-deps.diff

```diff
@@ -0,0 +1,20 @@
+diff -Nurd eccodes-2.13.0-Source.orig/CMakeLists.txt eccodes-2.13.0-Source/CMakeLists.txt
+--- eccodes-2.13.0-Source.orig/CMakeLists.txt	2019-07-08 14:06:44.000000000 +0200
++++ eccodes-2.13.0-Source/CMakeLists.txt	2019-07-19 13:27:24.158310812 +0200
```

Comment:
Two (optional) minor requests:
* Could you please report this upstream and post a link to the bug report? The patch itself would probably need some modifications to become more general and suitable for upstream, but it's ok for MacPorts only.
* Any chance to use just the following two lines and remove the `-p1`?
```
--- CMakeLists.txt
+++ CMakeLists.txt
```

Reply:
Since you prefer, I'll remove the directory name. But you'll also understand that have two trees next to each other, a modified one and an original one, and then run `diff -Nurd` is very convenient. It will then put the directory name in the patch file. Maybe there is some magical `diff` option to accomplish that.